brain-marks/Settings/Views/SettingsView.swift

```diff
@@ -19,11 +20,25 @@ struct SettingsView<ViewModel: InfoViewModel>: View {
             VStack {
                 Form {
                     
-                    Section {
+                    Section(header: Text("Appearance")) {
```

Comment:
Can you move the whole section under the List of Contributors part, but before the Links section?